TypeCobol.LanguageServer/TypeCobolCustomLanguageServer/TypeCobolCustomLanguageServer.cs

```diff
@@ -189,7 +189,7 @@ protected virtual void OnDidReceiveSignatureHelpContext(SignatureHelpContextPara
         /// <param name="bForced">Force the server to send the program OutlineNodes</param>
         protected virtual void OnDidReceiveRefreshOutline(string uri, bool bForced)
         {
-            var context = GetDocumentContextFromStringUri(uri, false);
+            var context = GetDocumentContextFromStringUri(uri, Workspace.SyntaxTreeRefreshLevel.RebuildNodes);
```

Comment:
Why not 
```suggestion
            var context = GetDocumentContextFromStringUri(uri, Workspace.SyntaxTreeRefreshLevel.NoRefresh);
```
?

Reply:
I thought it was a bug because the outline needs Nodes to be displayed correctly, no ?

Reply:
Yes indeed, I thought about the case when the server send an update of the Outline to the client.
But as I unterdstand it, this case is when the client want to refresh the whole Outline.

Reply:
Well i'm not sure now 🤨, because the `OnDidReceiveRefreshOutline` method is called in two spots:
- as an event handler when the ProgramClassDocument has changed
- as part of `OnDidSaveTextDocument` method

So it is not a notification/request method but an internal mechanism to refresh the outline data and then sends the data to the client. But it is still unclear for me if we need to rebuild nodes... Also why call this method in the DidSaveTextDocument process if we already have a ProgramClass refresh scheduled ?

Reply:
Will have to be fixed along with #1818.